packages/web3/test/e2e/get_transaction_from_block.test.ts

```diff
@@ -0,0 +1,115 @@
+				// to: null,
+				// value: '0x0',
+				// yParity: '0x0'
```

Comment:
Created #5997 to track this, but not sure what we should do

Reply:
There should be support of Nethermind as well along with Geth in our lib. For read JSON RPC calls we can manage it in our library to be consistent with `input` field. 
![image](https://user-images.githubusercontent.com/86780488/232801473-2818a399-1ae4-4bbd-929f-1df020ab1c86.png)